### PR TITLE
move logout event to service

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3365689&stafilter=NotComplete&fileview=grid

Signed-off-by: romanetar <roman_ag@hotmail.com>

### CODE REVIEW

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: app/libs/Auth/AuthService.php [hunk 1]
```diff
@@ -267,6 +277,13 @@ public function loginWithOTP(OAuth2OTP $otpClaim, ?Client $client = null, bool $
     public function logout()
     {
         Log::debug("AuthService::logout");
```
[the remark this answers is not in the document]
Comment (romanetar) [reply app/libs/Auth/AuthService.php]: done

**Comment by romanetar** [comment]
@smarcet review comments addressed

**Review by smarcet — APPROVED** [review]
LGTM

**Pull request merged**